composer.json

```diff
@@ -63,6 +77,7 @@
         "symfony/asset": "^3.4|^4.3",
         "symfony/config": "^3.4|^4.3",
         "symfony/console": "^3.4|^4.3",
+        "symfony/debug-bundle": "^3.4|^4.3",
```

Comment:
This bundle is only used in dev / test environment, it shouldn't be required for consumers of `sylius/sylius` package.